corehq/apps/consumer_user/tests/test_views.py

```diff
@@ -140,10 +142,17 @@ class LoginTestCase(TestCase):
 
     def setUp(self):
         super().setUp()
-        self.login_url = reverse('consumer_user:patient_login')
-        self.homepage_url = reverse('consumer_user:patient_homepage')
+        self.login_url = reverse('consumer_user:consumer_user_login')
+        self.homepage_url = reverse('consumer_user:consumer_user_homepage')
         self.client = Client()
 
+    def tearDown(self):
+        super().tearDown()
```

Comment:
`super().tearDown()` should happen at the end of this method.

Reply:
Fixed it